eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
@gggeek It holds the actual property name "resolved" dynamically, so in this case it would mean
`$this->defaultLanguageCode`
but in case of other ValueObjects, like `ContentType` it would be resolved as:
`$this->mainLanguageCode`

This confusion is the main reason I'm rather against making dynamically accessed properties. Moreover it confuses IDE and makes code maintenance more difficult.

However it's ValueObject which by definition uses dynamic properties and also it's internal, so in this case I'm fine with it as long as this is what @andrerom implied :)